templates/contracts/contract_create.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
```

Comment:
This template file could be improved following custom form rendering that we did in other places. For example
```
# templates/contracts/contract_create.html
<form method="post">
    {% csrf_token %}
    {{ form.non_field_errors }}
    <div class="form-group">
        {{ form.unit.errors }}
        {{ form.unit.label_tag }}
        {{ form.unit }}
    </div>
    <div class="form-group">
        {{ form.tenant.errors }}
        {{ form.tenant.label_tag }}
        {{ form.tenant }}
    </div>
    <!-- Add similar blocks for other fields -->
    <div class="d-grid gap-2">
        <button type="submit" class="btn btn-primary">Create Contract</button>
    </div>
</form>

```